web/js/MapUI/Components/BufferQuickAnimate/BufferQuickAnimate.js

```diff
@@ -0,0 +1,92 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { getNumberStepsBetween, getNextDateTime } from '../../../modules/date/util';
+import { getSelectedDate } from '../../../modules/date/selectors';
+import { promiseImageryForTime } from '../../../modules/map/util';
+import { setPreload } from '../../../modules/date/actions';
+
+const BufferQuickAnimate = (props) => {
+  const {
+    action,
+    date,
+    dateCompareState,
+    lastPreloadDate,
+    preloaded,
+    promiseImageryState,
+    setPreload,
```

Comment:
"setPreload" is imported at line 7 & also assigned a value here. Which one do we need?

Reply:
This is just a quirk of naming Redux actions. I am importing the action on line 7 from our date action creator file. Then in `mapDispatchToProps` I create a function prop that dispatches this action which I then have to list in the props to be able to use it in the component. While you can technically set the name as the action to something else, I find that it is less confusing to just use the same name, which is mostly what we do throughout all of our components.